pkg/operator/certmonitor/util.go

```diff
@@ -0,0 +1,29 @@
+package certmonitor
+
+import (
+	"k8s.io/api/core/v1"
+)
+
+func SetCertificateMonitoredConfigMap(config *v1.ConfigMap, certificateType CertificateType) {
```

Comment:
Why you want to make them public, if the certrotation is the only one using them let's make those private and live inside certrotation controller?

Reply:
If left public, service-ca could use these since ATM it does not utilize the certrotation controller for its certs.

Reply:
Is there any reason service-ca can't use the certrotation mechanism, I'd be more interested in aligning the functionality rather than multiplying options.

Reply:
Although it looks like that would need a public metrics collector controller as well. @mfojtik would it be possible to make it so service-ca can use this for cert metrics outside of a certrotation controller?

Reply:
> Is there any reason service-ca can't use the certrotation mechanism, I'd be more interested in aligning the functionality rather than multiplying options.

I'll look and see if there might be any problems using it. The service-ca operation is a single CA with N number of server certs, with no client certs. So, I'm not sure if the usage pattern is an exact fit.

Reply:
Honestly I'd prefer we share as much code as possible rather than having similarities in two places. It's better for all of us if we have the same foundation that we're working on :smiley:

Reply:
There is nothing preventing people for using this to report their cert metrics if they follow the annotations/labels.